runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -279,6 +298,22 @@ public int getItemPriceWithSource(int itemID, boolean useWikiPrice)
 		{
 			return 1000;
 		}
+		if (itemID == MASTER_SCROLL_BOOK)
+		{
+			// Can only hold 13k tradeable teleport scrolls, unlikely this overflows an int
+			// as each scroll would need to be over 165k ea
```

Comment:
I guess this must have changed since the time of writing. There are now 15 types of scrolls, making the amount ~143k each.

Reply:
Watson and Keymaster teleports can't be traded. Arguably I suppose they shouldn't even be in the map then since their price will always be 0?

Reply:
As was mentioned on Discord, only Keymaster teleports are untradeable, so I suppose it would be fine to leave them out. (maybe include a comment explaining that)